packages/types/src/derived.ts

```diff
@@ -1,9 +1,16 @@
+/* eslint-disable */
```

Comment:
I've added it because otherwise eslint is forcing me to add JSDocs for the `state`, `actions` and `libraries` which in my opinion actually makes it harder to read what the type is.